app/ts/store/account.ts

```diff
@@ -15,6 +15,7 @@ type AccountConnected = {
 type AccountDisconnected = {
 	status: 'disconnected'
 	connect: () => void
+	ensureConnected: () => void
```

Comment:
The implementation for both of these are async so the interface should be async as well.  The caller can decide not to await if they like, but they should have the option to.
```suggestion
	connect: () => Promise<void>
	ensureConnected: () => Promise<void>
```